packages/plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -0,0 +1,143 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+ /* eslint camelcase: [0] */
+
+import '../..';
+import CiscoSpark from '@ciscospark/spark-core';
+import {assert} from '@ciscospark/test-helper-chai';
+import testUsers from '@ciscospark/test-helper-test-users';
+import uuid from 'uuid';
+
+const moment = require(`moment`);
+
+describe(`plugin-calendar`, function() {
+  describe(`#meetings()`, () => {
+
+    let createdMeeting, mccoy, organizer, spark, spock;
+
+    before(`create test users`, () => testUsers.create({
+      count: 3,
+      config: {
+        entitlements: [
+          `spark`,
+          `squaredCallInitiation`,
+          `squaredInviter`,
+          `squaredRoomModeration`,
+          `webExSquared`,
+          `squaredFusionCal`
+        ]
+      }
+    })
+      .then((users) => {
+        [organizer, spock, mccoy] = users;
+
+        spark = new CiscoSpark({
+          credentials: {
+            authorization: organizer.token
+          }
+        });
+      }));
+
+    const fromInterval = moment.utc().startOf(`day`).subtract(1, `week`).format(`YYYY-MM-DD[T]HH:mm:ss.SSS[Z]`);
```

Comment:
Do we really need moment for this? If the api can't accept either epoch timestamps or `Date#toISOString()` strings, we have a problem.